react/components/Account/README.md

```diff
@@ -16,6 +18,17 @@ You can use it in your code like a React component with the jsx tag: `<ContentWr
 </ContentWrapper>
 ```
 
+
+| Prop name          | Type       | Description                                                                 |
```

Comment:
I think it's better to not write out these spaces, it makes it harder to change later.